deployment/supervisor-adage.conf

```diff
@@ -0,0 +1,8 @@
+command=/home/ubuntu/.venv/adage/bin/gunicorn adage.wsgi:application --bind 127.0.0.1:8001 --error-logfile /var/log/adage-gunicorn.log -w 3
+directory=/home/ubuntu/py3-adage-backend/adage/
+user=ubuntu
+group=ubuntu
```

Comment:
Good question! I was lazy when deploying the test server on AWS. The main reason why I set these two lines to `ubuntu` was because `nobody` group doesn't exist on Ubuntu 18.04 now. 

After digging a little deeper on Ubuntu users and groups, I found that `nobody` is now a member of `nogroup` group, so I will change user to `nobody` and group to `nogroup`.

Reply:
Ah.. good find!

Reply:
This supervisor configuration now at least follows the example from the gunicorn deployment page: https://docs.gunicorn.org/en/stable/deploy.html